DOC-104 Redazione del verbale esterno 23-11-23

ZacconeRosario opened the pull request:
Link alla issue di Jira: https://error418swe.atlassian.net/browse/DOC-104?atlOrigin=eyJpIjoiNGU1YzhhNzE2MDMxNDNjMTg4NDA2MjBhMDI5OGFmMGEiLCJwIjoiaiJ9

Note: n.d.

### ✏️ Checklist

- [x] documento testato;
- [x] titolo della PR conforme;
- [x] designato almeno un verificatore.

ZacconeRosario committed:
Redazione del verbale esterno 23-11-23

actions-user committed:
Aggiunto changelog vuoto

giogardin commented on 2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ:
```diff
@@ -0,0 +1,77 @@
+= Ordine del giorno
+- Presentazione, supportata da una board Miro, delle assunzioni fatte dal team durante il periodo di individuazione degli use case;
+
+- Presentazione, supportata da una board Miro, delle domande raccolte dal team rigurdo l'analisi dei requisiti. Le domande si articolano in:
```
- Typo: riguardo
- Analisi dei Requisiti e non "analisi dei requisiti"

giogardin commented on 2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ:
```diff
@@ -0,0 +1,77 @@
+);
+
+= Ordine del giorno
+- Presentazione, supportata da una board Miro, delle assunzioni fatte dal team durante il periodo di individuazione degli use case;
```
Assunzioni purtroppo è un false friend di assumptions. La parola corretta è ipotesi o supposizioni

giogardin commented on 2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ:
```diff
@@ -0,0 +1,77 @@
+- continuazione del lavoro di stesura degli use case;
+- organizzazione di un meeting interno post meeting esterno per riflettere meglio sul lavoro da fare e sulla sua suddivisione.
+
+= Firme partecipanti esterni
```
Questo paragrafo non è più necessario, lo puoi rimuovere

giogardin reviewed (commented):
Ottimo lavoro! Ci sono solo delle piccole cose da sistemare

actions-user committed:
Aggiunti dettagli di rilascio

giogardin commented on 2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ:
```diff
@@ -0,0 +1,77 @@
+\
+
+== Richieste all'azienda proponente riguardo materiale utile durante il progetto
+Il team ha avanzato la richiesta di materiale e informazioni come le le caratteristiche dei prodotti, esempi di codifica di scaffali e bin ed esempi di piantina di magazzino. Le proposte sono state accolte dal referente aziendale, che ci ha assicurato il soddisfacimento delle nostre richieste.
```
"le le"

actions-user committed:
Aggiunti dettagli di rilascio

giogardin commented on 2 - RTB/Documentazione interna/Verbali/Esterni/23-11-23/23-11-23.typ:
```diff
@@ -0,0 +1,77 @@
+
+== Presentazione delle assunzioni fatte
+Il team ha iniziato il meeting presentando delle assunzioni al referente aziendale, per avere un riscontro riguardo la loro validità.\
+La prima assunzione specificava la presenza di un solo prodotto per ogni bin, ed è stata confermata e ritenuta una semplificazione utile. Per aree di carico/scarico questa assunzione non è valida, ma per semplicità di sviluppo si può tenere questa semplificazione.\
```
Altra cosa che farei: trasformare questa lista in un elenco numerato (in Typst si crea come una lista puntata ma usando `+`)

actions-user committed:
Aggiunti dettagli di rilascio

ZacconeRosario committed:
Update 23-11-23.typ

Corretti alcuni typo e sostituito il termine "assunzioni" con "ipotesi"

ZacconeRosario committed:
Update 23-11-23.typ

Rimozione paragrafo non necessario (firme partecipanti esterni)

ZacconeRosario committed:
Update 23-11-23.typ

Corretti ulteriori typo, aggiunto elenco numerato per la lista delle ipotesi presentate

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio

actions-user committed:
Aggiunti dettagli di rilascio
